### ISSUE TITLE
[4.0] Atum admin menu accessibility regression

### ISSUE THREAD

**Issue by brianteeman** [joomla/joomla-cms#14758]
### Steps to reproduce the issue
Install j4 and try to navigate through the left hand (main) menu using the keyboard

See movie - first part shows mouse navigation - second part shows failing keyboard navigation
![atum-menu](https://cloud.githubusercontent.com/assets/1296369/24080480/f986f49e-0c97-11e7-8e2d-6df42a7a82dc.gif)

**Comment by C-Lodder** [comment]
Known issue at the moment.

We're still waiting for surveys to be send out and get the results before we refactor the sidebar.

**Comment by armenos** [comment]
Hallo Brian,
I've seen this issue two days ago and I'm making a list now with all issue about mainmenu in back-end

**Comment by armenos** [comment]
I think this one can be closes here and we can work on this here #14812 
There is e new issue made by me about mainmenu accessibility joomla 4.0.0-dev over here #14812 <hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/14758">issues.joomla.org/tracker/joomla-cms/14758</a>.</sub>

**Comment by brianteeman** [comment]
Closing for now see #14812
